evm_body.yaml

```diff
@@ -1572,6 +1709,18 @@ BlockTag:
     - earliest
   default: latest
 
+BlockTag_polygon_zkevm:
+  title: Block tag
+  type: string
+  description: |
+    One of the following **block tags**:
+      * `latest` - The most recent block in the canonical chain observed by the client, this block may be re-orged out of the canonical chain even under healthy/normal conditions.
+      * `earliest` - The lowest numbered block the client has available. Intuitively, you can think of this as the first block created.
```

Comment:
is these are the only blog tag options we should update the other blog tag params above?

Reply:
Yes @elanh these are the only ones available here, you can verify here: 

https://github.com/0xPolygonHermez/zkevm-node/blob/develop/jsonrpc/endpoints_zkevm.openrpc.json#L251-L259